File: BOJ/1439.py
s = input()
res = 0
chk = s[0]
pq = [0, 0]
pq[int(chk)] += 1

for i in range(len(s)):
    if chk != s[i]:
        res += 1
        chk = s[i]
        pq[int(chk)] += 1

print(min(pq))

File: BOJ/1439.js
let input = require("fs")
  .readFileSync("예제.txt")
  .toString()
  .trim()
  .split("\n");

let chk = input[0][0];
let arr = [0, 0];
arr[Number(chk)] += 1;

for (let i = 0; i < input[0].length; i++) {
  if (chk !== input[0][i]) {
    chk = input[0][i];
    arr[Number(chk)]++;
  }
}

console.log(Math.min(...arr));
